core/src/epicli/data/aws/defaults/infrastructure/virtual-machine.yml

```diff
@@ -223,14 +223,14 @@ specification:
        destination_port_range: "22"
        source_address_prefix: "0.0.0.0/0"
        destination_address_prefix: "0.0.0.0/0"
-     - name: node_exporter
-       description: Allow node_exporter traffic
+     - name: repository
```

Comment:
Seems you replaced the nope_exporter entry with this one. Shouldn't there be both?

Reply:
Need to fix.

Reply:
Fixed.